<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuImport">
     <property name="title">
      <string>Import</string>
     </property>
     <addaction name="actionMIDI"/>
    </widget>
    <widget class="QMenu" name="menuExport_as">
     <property name="title">
      <string>Export as...</string>
     </property>
     <addaction name="actionMIDI_2"/>
    </widget>
    <widget class="QMenu" name="menuRecent_files">
     <property name="title">
      <string>Recent files</string>
     </property>
    </widget>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="menuRecent_files"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="menuImport"/>
    <addaction name="menuExport_as"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuPlay">
    <property name="title">
     <string>Play</string>
    </property>
    <addaction name="actionPlay"/>
    <addaction name="actionStop"/>
    <addaction name="actionBack"/>
    <addaction name="actionRepeat"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionEdit_Mode"/>
    <addaction name="actionGrid_Snap"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuPlay"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="../res/qtau.qrc">
     <normaloff>:/images/b_blank.png</normaloff>:/images/b_blank.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../res/qtau.qrc">
     <normaloff>:/images/b_folder.png</normaloff>:/images/b_folder.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../res/qtau.qrc">
     <normaloff>:/images/b_save.png</normaloff>:/images/b_save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="../res/qtau.qrc">
     <normaloff>:/images/b_power.png</normaloff>:/images/b_power.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="icon">
    <iconset resource="../res/qtau.qrc">
     <normaloff>:/images/b_play.png</normaloff>:/images/b_play.png</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="../res/qtau.qrc">
     <normaloff>:/images/b_stop.png</normaloff>:/images/b_stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Space</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset resource="../res/qtau.qrc">
     <normaloff>:/images/b_save.png</normaloff>:/images/b_save.png</iconset>
   </property>
   <property name="text">
    <string>Save as...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="../res/qtau.qrc">
     <normaloff>:/images/b_undo.png</normaloff>:/images/b_undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="../res/qtau.qrc">
     <normaloff>:/images/b_redo.png</normaloff>:/images/b_redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="../res/qtau.qrc">
     <normaloff>:/images/b_cross.png</normaloff>:/images/b_cross.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionBack">
   <property name="icon">
    <iconset resource="../res/qtau.qrc">
     <normaloff>:/images/b_play_prev.png</normaloff>:/images/b_play_prev.png</iconset>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
   <property name="shortcut">
    <string>Backspace</string>
   </property>
  </action>
  <action name="actionRepeat">
   <property name="icon">
    <iconset resource="../res/qtau.qrc">
     <normaloff>:/images/b_repeat.png</normaloff>:/images/b_repeat.png</iconset>
   </property>
   <property name="text">
    <string>Repeat</string>
   </property>
   <property name="shortcut">
    <string>Shift+Space</string>
   </property>
  </action>
  <action name="actionGrid_Snap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/qtau.qrc">
     <normaloff>:/images/b_ruler.png</normaloff>:/images/b_ruler.png</iconset>
   </property>
   <property name="text">
    <string>Grid Snap</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionEdit_Mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/qtau.qrc">
     <normaloff>:/images/b_pencil.png</normaloff>:/images/b_pencil.png</iconset>
   </property>
   <property name="text">
    <string>Edit Mode</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionMIDI">
   <property name="text">
    <string>MIDI</string>
   </property>
  </action>
  <action name="actionMIDI_2">
   <property name="text">
    <string>MIDI</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../res/qtau.qrc"/>
 </resources>
 <connections/>
</ui>
